### PR TITLE
system_tables: add column description for several tables

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add column description for several information_schema tables:

- COLLATIONS
- COLLATION_CHARACTER_SET_APPLICABILITY
- CHARACTER_SETS
- SESSION_VARIABLES
- TABLES
- TIDB_HOT_REGIONS

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3617, https://github.com/pingcap/docs-cn/pull/3678
- Other reference link(s):

### CODE REVIEW

File: system-tables/system-table-information-schema.md [hunk 1]
```diff
@@ -679,6 +677,30 @@ SHOW TABLES
...
+* `AVG_ROW_LENGTH`: The average row length of the table. `AVG_ROW_LENGTH` = `DATA_LENGTH` / `TABLE_ROWS`.
+* `DATA_LENGTH`: Data length. `DATA_LENGTH` = `TABLE_ROWS` * the sum of storage lengths of the columns in the tuple. The replicas of TiKV is not taken into account.
+* `MAX_DATA_LENGTH`: The maximum data length. The value is currently `0`, which means the data length has no upper limit.
+* `INDEX_LENGTH`: The index length. `INDEX_LENGTH` = `TABLE_ROWS` * the sum of lengths of the columns in the index tuple. The replicas of TiKV is not taken into account.
+* `DATA_FREE`: Data fragment. The value is currently `0`.
+* `AUTO_INCREMENT`: The current step of the auto-increment primary key.
+* `CREATE_TIME`: The time at which the table is created.
+* `UPDATE_TIME`: The time at which the table is updated.
+* `CHECK_TIME`: The time at which the table is checked.
+* `TABLE_COLLATION`: The character check code set of the table.
```
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:699]: I'm not sure of this translation. Still waiting for the author's reply.

File: system-tables/system-table-information-schema.md [hunk 2]
```diff
@@ -679,6 +677,30 @@ SHOW TABLES
...
+* `AVG_ROW_LENGTH`: The average row length of the table. `AVG_ROW_LENGTH` = `DATA_LENGTH` / `TABLE_ROWS`.
+* `DATA_LENGTH`: Data length. `DATA_LENGTH` = `TABLE_ROWS` * the sum of storage lengths of the columns in the tuple. The replicas of TiKV is not taken into account.
+* `MAX_DATA_LENGTH`: The maximum data length. The value is currently `0`, which means the data length has no upper limit.
+* `INDEX_LENGTH`: The index length. `INDEX_LENGTH` = `TABLE_ROWS` * the sum of lengths of the columns in the index tuple. The replicas of TiKV is not taken into account.
+* `DATA_FREE`: Data fragment. The value is currently `0`.
+* `AUTO_INCREMENT`: The current step of the auto-increment primary key.
+* `CREATE_TIME`: The time at which the table is created.
+* `UPDATE_TIME`: The time at which the table is updated.
+* `CHECK_TIME`: The time at which the table is checked.
+* `TABLE_COLLATION`: The collation of strings in the table.
```
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:699]: Confirmed with the author on the translation of `TABLE_COLLATION`

File: system-tables/system-table-information-schema.md [hunk 3]
```diff
@@ -774,6 +796,19 @@ desc TIDB_HOT_REGIONS;
 +----------------+---------------------+------+-----+---------+-------+
 ```
 
+The description of columns in the `TIDB_HOT_REGIONS` table is as follows:
+
+* `TABLE_ID`: ID of the table which the hot Region is located.
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:801]:
```suggestion
* `TABLE_ID`: ID of the table in which the hot Region is located.
```

File: system-tables/system-table-information-schema.md [hunk 4]
```diff
@@ -774,6 +796,19 @@ desc TIDB_HOT_REGIONS;
...
+* `INDEX_ID`: ID of the index which the hot Region is located.
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:802]: ditto

File: system-tables/system-table-information-schema.md [hunk 5]
```diff
@@ -774,6 +796,19 @@ desc TIDB_HOT_REGIONS;
...
+* `DB_NAME`: The database name of the object which the hot Region is located.
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:803]: ditto

File: system-tables/system-table-information-schema.md [hunk 6]
```diff
@@ -774,6 +796,19 @@ desc TIDB_HOT_REGIONS;
...
+* `TABLE_NAME`: The name of the table which the hot Region is located.
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:804]: ditto

File: system-tables/system-table-information-schema.md [hunk 7]
```diff
@@ -774,6 +796,19 @@ desc TIDB_HOT_REGIONS;
...
+* `INDEX_NAME`: The name of the index which the hot Region is located.
```
Comment (TomShawn) [inline system-tables/system-table-information-schema.md:805]: ditto

File: system-tables/system-table-information-schema.md (hunk 4 above)
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:802]:
```suggestion
* `INDEX_ID`: ID of the index in which the hot Region is located.
```

File: system-tables/system-table-information-schema.md (hunk 5 above)
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:803]:
```suggestion
* `DB_NAME`: The database name of the object in which the hot Region is located.
```

File: system-tables/system-table-information-schema.md (hunk 6 above)
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:804]:
```suggestion
* `TABLE_NAME`: The name of the table in which the hot Region is located.
```

File: system-tables/system-table-information-schema.md (hunk 7 above)
Comment (ran-huang) [inline system-tables/system-table-information-schema.md:805]:
```suggestion
* `INDEX_NAME`: The name of the index in which the hot Region is located.
```

**Comment by TomShawn** [comment]
LGTM
@AilinKid @bb7133 PTAL

**Comment by ran-huang** [comment]
@TomShawn PTAL again. Thanks 🙏

File: information-schema/information-schema-collation-character-set-applicability.md [hunk 8]
```diff
@@ -38,3 +38,13 @@ SELECT * FROM collation_character_set_applicability WHERE character_set_name='ut
 +----------------+--------------------+
 1 row in set (0.00 sec)
 ```
+
++----------------+--------------------+
+| COLLATION_NAME | CHARACTER_SET_NAME |
+| utf8mb4_bin | utf8mb4 |
++----------------+--------------------+
```
Comment (yikeke) [inline information-schema/information-schema-collation-character-set-applicability.md:45]: lines 42 to 45 are duplicate with lines 34 to 38. Removed duplicates in https://github.com/pingcap/docs/pull/3470/commits/520218547d2599a300960c01109c676a02f4c640. (also for [the Chinese version](https://github.com/pingcap/docs-cn/pull/4109/files#diff-6d4157f625dd3f18efcbc5add886ad6cR34-R39))

**Pull request merged**